src/ui_ng/lib/src/helm-chart/chart-detail/chart-detail.component.scss

```diff
@@ -10,11 +12,5 @@
 }
 
 .detail-loading {
-    position: absolute;
-    top: 0;
-    left: 0;
-    right:0;
-    bottom:0;
-    width: 108px !important;
-    height: 108px !important;
+    @include absolute-center;
 }
```

Comment:
if use absolute-center directly, the default size will be 108px, will it be too big?

Reply:
I think the size is OK
![image](https://user-images.githubusercontent.com/1069785/43716870-3793b00c-99b8-11e8-9e25-f4a420ada40b.png)